ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/outpost/day2/provision/cucushift-installer-rehearse-aws-ipi-outpost-day2-provision-chain.yaml

```diff
@@ -0,0 +1,48 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-outpost-day2-provision
+  steps:
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: aws-provision-vpc-shared
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-aws-custom-vpc
+  - ref: ipi-conf-aws
+  - ref: ipi-install-hosted-loki
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-install-install-aws
+  - ref: ipi-install-times-collection
+  - ref: aws-provision-tags-for-byo-vpc
+  - ref: enable-qe-catalogsource
+  - ref: storage-conf-storageclass-set-default-storageclass
+  - chain: network-mtu-migration-decrease
+  - ref: edge-nodes-machineset-manifests
+  - ref: edge-nodes-provision-localzone-day2
```

Comment:
I am thinking if it is possible to make the above two step to be simple as much as possible for a dedicated purpose, to reduce complexity because of compatibility for different scenarioes (localzone, wavelength, and outpost) .